### PR TITLE
Feature/8025 8248 UI tests screenshots

### PR DESCRIPTION
**Pull request by SwissMountainFrog**
This PR addes the following certification validation screenshots to the list
Ticket https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-8489

CertValScreenshots

### CODE REVIEW

**Review by 30mar — APPROVED** [review]

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift [hunk 1]
```diff
@@ -36,6 +36,20 @@ class HealthCertificateService {
...
+						testEntries: [TestEntry.fake(dateTimeOfSampleCollection: "2021-04-12T16:01:00Z")]
+					),
+					and: CBORWebTokenHeader.fake()
+				)
+				
+				if case let .success(base45) = dcc {
+					registerHealthCertificate(base45: base45)
+				}
+				
+			}
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift:52]: I don't think we need this, do we?

File: src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift [hunk 2]
```diff
@@ -322,11 +324,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
 		return VaccinationValueSetsProvider(client: CachingHTTPClient(), store: store)
 	}
 	
-	private lazy var healthCertificateValidationService: HealthCertificateValidationProviding = HealthCertificateValidationService(
-		store: store,
-		client: client,
-		vaccinationValueSetsProvider: vaccinationValueSetsProvider
-	)
+
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift:327]:
```suggestion
```

I think you got two empty lines here

File: src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift [hunk 3]
```diff
@@ -322,11 +324,34 @@ class AppDelegate: UIResponder, UIApplicationDelegate, CoronaWarnAppDelegate, Re
...
+			return mock
+		}
+		#endif
+		
+		let service = HealthCertificateValidationService(
+			store: store,
+			client: client,
+			vaccinationValueSetsProvider: vaccinationValueSetsProvider
+		)
+		return service
```
Comment (nickguendling) [inline src/xcode/ENA/ENA/Source/AppDelegate & Globals/AppDelegate.swift:353]:
```suggestion
		return HealthCertificateValidationService(
			store: store,
			client: client,
			vaccinationValueSetsProvider: vaccinationValueSetsProvider
		)
```

Just to make it a little simpler 🤓

**Review by nickguendling — CHANGES_REQUESTED** [review]

File: src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift (hunk 1 above)
Comment (SwissMountainFrog) [reply src/xcode/ENA/ENA/Source/Services/HealthCertificate/HealthCertificateService.swift]: Nope. Totale forgot to code clean-up this. Removed...!

**Review by nickguendling — APPROVED** [review]

**Review by KaiTeuber — APPROVED** [review]

**Pull request merged**